kythe/cxx/common/kzip_encoding.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef KYTHE_CXX_COMMON_KZIP_ENCODING_H_
+#define KYTHE_CXX_COMMON_KZIP_ENCODING_H_
+
+#include "absl/strings/string_view.h"
```

Comment:
x

Reply:
You shouldn't need this include or the dependency.